source/index.html.md

```diff
@@ -630,6 +630,31 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects. Currenlty we only
+accept the JWE.
+Your system must extract the content encryption key (<code>cek</code>) used to encrypt the JWE
+payload and Base64-encode it, along with the entire JWE.
+
+<dl class="dl-vertical">
+  <dt>samsungpay[jwe]</dt>
+  <dd>
+    Type: <a href="https://tools.ietf.org/html/rfc7516#section-3.1">JWE Compact serialization</a>
+    <br>
+    <i>Required</i>
+    <br>
+    Complete payment token as received from Samsung Pay.
+  </dd>
+  <dt>samsungpay[symmetric_key]</dt>
+  <dd>
+    Type: Base-64 encoded string
```

Comment:
Mention format to be something like `[:base64:]{16}`?

Reply:
Well, the Base64-string will be 28 characters long. But that's not enough, because 19, 20 and 21 bytes Base64-encoded are all 28 characters.

I'm experimenting with a less obtuse pseudo-regexp syntax `[:...:]`.

I'll give it some thought.

Reply:
I'll save that for another day, `[:base64:]` it is.

Reply:
I saw "Base64-encode" earlier in this PR and notice that "Base-64 encoded" is inconsistent with that.

Reply:
These 16 bytes will always end up as a 24 char long string when Base64 encoded, right?

If I am not mistaken, let's be stringent and write `[:base64:]{24}` 🙂

Reply:
Sure.